Makefile

```
@@ -1,4 +1,5 @@
 REF = main
+LATEST := $(shell python3 -c 'import json; versions = json.load(open("versions.json")); versions.sort(); print(versions[-1])')
```

1. Use `=` instead of `:=` so make expands this when it's used instead of when it's defined.
2. We use the default value for https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#lastVersion, and that takes the first version from versions.json, so we should do the same here.
```suggestion
LATEST = $(shell jq -r '.[0]' versions.json)
```